hathor/transaction/transaction.py

```diff
@@ -271,27 +280,48 @@ def get_complete_token_info(self) -> dict[TokenUid, TokenInfo]:
     def _get_token_info_from_inputs(self) -> dict[TokenUid, TokenInfo]:
         """Sum up all tokens present in the inputs and their properties (amount, can_mint, can_melt)
         """
-        token_dict: dict[TokenUid, TokenInfo] = {}
-
-        default_info: TokenInfo = TokenInfo(0, False, False)
 
         # add HTR to token dict due to tx melting tokens: there might be an HTR output without any
         # input or authority. If we don't add it, an error will be raised when iterating through
         # the outputs of such tx (error: 'no token creation and no inputs for token 00')
-        token_dict[self._settings.HATHOR_TOKEN_UID] = TokenInfo(0, False, False)
+        token_dict: dict[TokenUid, TokenInfo] = {
+            self._settings.HATHOR_TOKEN_UID: TokenInfo(0, False, False, None, [], [])}
```

Comment:
Maybe refactor this `empty_token_info` into a class method `TokenInfo.create_empty()`?